src/tests/ftest/util/dmg_utils.py

```diff
@@ -1,6 +1,6 @@
 """
-  (C) Copyright 2018-2024 Intel Corporation.
-  (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+  (C) Copyright 2018-2025 Intel Corporation.
+  (C) Copyright 2025 Hewlett Packard Enterprise Development LP.
```

Comment:
Unnecessary copyright update now

Reply:
If you manually revert (no changes to the file) the githook should not add it back in. But if it does, you can commit with `--no-verify`